src/encoded/types/analysis.py

```diff
@@ -299,7 +301,26 @@ def superseded_by(self, request, superseded_by):
         "type": "array",
         "notSubmittable": True,
         "items": {
-            "type": "object"
+            "type": "object",
+            "properties": {
+                "biological_replicates":{
+                    "type": "array",
+                    "items": {
+                        "type": "integer",
+                    }
+                },
+                "files": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "linkTo": "File"
+                    }
+                },
+                "quality_metric": {
+                    "type": "string",
```

Comment:
I think have to return `@id` of quality metric now instead of object (`qm_obj`).

Reply:
done